pacman/operations/rigged_algorithms/random_placer.py

```diff
@@ -0,0 +1,97 @@
+from pacman.utilities.utility_objs import ResourceTracker
+from pacman.utilities.algorithm_utilities import placer_algorithm_utilities
+from pacman.model.placements import Placement, Placements
+from pacman.model.constraints.placer_constraints import SameChipAsConstraint
+
+from spinn_utilities.progress_bar import ProgressBar
+
+import random as default_random
+
+
+class RandomPlacer(object):
+
+    def __call__(self, machine_graph, machine):
+
+        # check that the algorithm can handle the constraints
+        ResourceTracker.check_constraints(machine_graph.vertices)
+
+        placements_copy = Placements()
```

Comment:
why placements copy? there's no copy here. its not even able to make a copy, its a placer!